source/dofs/dof_tools_constraints.cc

```diff
@@ -1070,6 +1101,1290 @@ namespace DoFTools
+              for (unsigned int line = 0;
+                   line < GeometryInfo<dim>::lines_per_face;
+                   ++line)
+                if (this_face->line(line)->vertex_index(0) >
+                    this_face->line(line)->vertex_index(1))
+                  direction_mother[line] = true;
+
+              // get the orientation from the intern edges of the children
+              std::vector<bool> direction_child_intern(
+                n_internal_lines_on_children, false);
+
+              // get the global vertex index of vertex in the center;
+              // we need this vertex index, to compute the direction
+              // of the internal edges
+              unsigned int center = this_face->child(0)->vertex_index(3);
+
+              // compute the direction of the internal edges
+              for (unsigned int line = 0; line < n_internal_lines_on_children;
+                   ++line)
+                if (line % 2 == 0)
+                  {
+                    direction_child_intern[line] =
+                      this_face->line(line)->child(0)->vertex_index(1) <
+                          center ?
+                        false :
+                        true;
+                  }
+                else
+                  {
+                    direction_child_intern[line] =
+                      this_face->line(line)->child(0)->vertex_index(1) >
+                          center ?
+                        false :
+                        true;
+                  }
+
+              // compute the direction of the outer edges
+              std::vector<bool> direction_child(n_external_lines_on_children,
+                                                false);
+              for (unsigned int line = 0;
+                   line < GeometryInfo<dim>::lines_per_face;
+                   ++line)
+                {
+                  if (this_face->line(line)->child(0)->vertex_index(0) >
+                      this_face->line(line)->child(0)->vertex_index(1))
+                    direction_child[2 * line] = true;
+                  if (this_face->line(line)->child(1)->vertex_index(0) >
+                      this_face->line(line)->child(1)->vertex_index(1))
+                    direction_child[2 * line + 1] = true;
+                }
+
+
+              // Orientation - Faces:
+              bool              mother_flip_x  = false;
+              bool              mother_flip_y  = false;
+              bool              mother_flip_xy = false;
+              std::vector<bool> child_flip_x(n_children_per_face, false);
+              std::vector<bool> child_flip_y(n_children_per_face, false);
+              std::vector<bool> child_flip_xy(n_children_per_face, false);
+              const unsigned int
+                vertices_adjacent_on_face[GeometryInfo<dim>::vertices_per_face]
+                                         [2] = {{1, 2}, {0, 3}, {3, 0}, {2, 1}};
+
+              { // Mother
```

Comment:
I think we typically do not place comments in the same line as code, unless the comment is short enough for a single line (whereas you here have 2 lines, albeit they are referring to different things).
```suggestion
              {
                // Mother
```

Reply:
Okay, I moved all comments into separate lines.